python/tests/browserstack/local_binary.py

```diff
@@ -0,0 +1,43 @@
+import platform
+import urllib.request
+import os
+import sys
+import zipfile
+import stat
+
+class LocalBinary:
+    def __init__(self):
+        is_64bits = sys.maxsize > 2**32
+        osname = platform.system()
+        if osname == 'Darwin':
+            self.http_path = "https://www.browserstack.com/browserstack-local/BrowserStackLocal-darwin-x64.zip"
+        elif osname == 'Linux':
```

Comment:
its better to avoid strict comparisons when it comes to os names, a possible alternative might be
```suggestion
        elif 'linux' in osname.lower():
```